### DISCUSSION TITLE
CPU Exceptions (zh-CN)

### DISCUSSION THREAD

**Discussion by giscus** [phil-opp/blog_os#1325] [Post Comments (translated)]
# CPU Exceptions (zh-CN)

CPU异常在很多情况下都有可能发生，比如访问无效的内存地址，或者在除法运算里除以0。为了处理这些错误，我们需要设置一个 中断描述符表 来提供异常处理函数。在文章的最后，我们的内核将能够捕获 断点异常 并在处理后恢复正常执行。

https://os.phil-opp.com/zh-CN/cpu-exceptions/

**Comment by EveGlowLuna** [comment]
So when I actually run `cargo run` , part of the screen goes like this:
```bash
InterruptStackFrame{hem well.
It did not crash!ointer: VirtAdddr(
        0x2056a1,
),
```
Is there something wrong with me?
